WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -58,6 +58,20 @@ final class SettingsViewController: UIViewController {
     ///
     private var sites = [Yosemite.Site]()
 
+    /// Payment Gateway Account ResultsController
+    ///
+    private let gatewayAccountResultsController: ResultsController<StoragePaymentGatewayAccount> = {
+        let storageManager = ServiceLocator.storageManager
+        /// TODO - resolve how I want to manage the siteID here. Should I just fetch all gatewayAccounts and filter by siteID later perhaps?
```

Comment:
I'm curious, what would be the benefit of fetching all accounts?

Reply:
Nah, it would be better to only return the gatewayAccount(s) for the site. I was just thinking out loud.

Reply:
In https://github.com/woocommerce/woocommerce-ios/pull/4381/commits/775566b5dcc26310f255e389af0a617e923a744c I removed this change - I'm deferring view controller changes to the last pull in this series.